src/screens/provider/ManageLocation.tsx

```diff
@@ -0,0 +1,190 @@
+            </h1>
+            <Form.Group className="row mb-3 col-md-8 " controlId="locationName">
+              <Form.Label>Name</Form.Label>
+              <Form.Control
+                onChange={(e) => {
+                  setLocation({ ...location, locationname: e.target.value })
+                }}
+                type="text"
+                placeholder="Enter Name"
+              />
+            </Form.Group>
+
+            <Form.Group
+              className="row mb-3 col-md-8 "
+              controlId="locationDescription"
+            >
+              <Form.Label>Description</Form.Label>
+              <Form.Control
+                onChange={(e) => {
+                  setLocation({ ...location, description: e.target.value })
+                }}
+                type="text"
+                placeholder="Enter Description"
+              />
+            </Form.Group>
+
+            <Form.Group
+              className="row mb-3 col-md-8 "
+              controlId="locationAddress"
+            >
+              <Form.Label>Address</Form.Label>
+              <Form.Control
+                onChange={(e) => {
+                  setLocation({ ...location, address: e.target.value })
+                }}
+                type="text"
+                placeholder="Enter Address"
+              />
+            </Form.Group>
+            <Form.Group className="row mb-3 col-md-8 " controlId="locationURL">
+              <Form.Label>Google Map URL</Form.Label>
+              <Form.Control
+                onChange={(e) => {
+                  setLocation({ ...location, url: e.target.value })
+                }}
+                type="text"
+                placeholder="Enter URL"
+              />
+            </Form.Group>
+            <Form.Group
+              className="row mb-3 col-md-8 "
+              controlId="locationOpenTime"
+            >
+              <Form.Label>Open Time</Form.Label>
+              <Form.Control
+                onChange={(e) => {
+                  setLocation({ ...location, openTime: e.target.value })
+                }}
+                type="time"
+              />
+            </Form.Group>
+
+            <Form.Group
+              className="row mb-3 col-md-8 "
+              controlId="locationCloseTime"
+            >
+              <Form.Label>Close Time</Form.Label>
+              <Form.Control
+                onChange={(e) => {
+                  setLocation({ ...location, closeTime: e.target.value })
+                }}
+                type="time"
+              />
+            </Form.Group>
+            <Form.Group className="col-md-8">
+              <label>Select available days for your location :</label>
+              <div className="row ">
+                {[
+                  'Monday',
+                  'Tuesday',
+                  'Wednesday',
+                  'Thursday',
+                  'Friday',
+                  'Saturday',
+                  'Sunday',
```

Comment:
We should do like this. Define the var for handle this info is better
```suggestion
                {[
                  'Monday',
                  'Tuesday',
                  'Wednesday',
                  'Thursday',
                  'Friday',
                  'Saturday',
                  'Sunday',
```